bigquery_etl/schema/__init__.py

```diff
@@ -17,6 +17,12 @@
 SCHEMA_FILE = "schema.yaml"
 
 
+class SchemaAssertError(Exception):
+    """Schema assert error."""
+
+    pass
```

Comment:
nit: definitely subjective, but I would just treat the string comment as a `pass`
```suggestion
```